src/middlewared/middlewared/plugins/enterprise/m_series_nvdimm_freebsd.py

```diff
@@ -59,20 +55,16 @@ def setup_m_series_nvdimm(self):
             self.DATA = result
 
             bios_dates = {
-                "TRUENAS-M40": datetime(2020, 2, 20),
-                "TRUENAS-M50": datetime(2020, 12, 3),
-                "TRUENAS-M60": datetime(2020, 12, 3),
+                "TRUENAS-M40": datetime(2020, 2, 20).date(),
```

Comment:
You can just use `from datetime import date` constructor

Reply:
Thanks!